zaza/openstack/charm_tests/ceph/rbd_mirror/tests.py

```diff
@@ -166,6 +184,18 @@ def setUpClass(cls):
         # get ready for multi-model Zaza
         cls.site_a_model = cls.site_b_model = zaza.model.get_juju_model()
         cls.site_b_app_suffix = '-b'
+        # Test if we have the needed Openstack applications.
```

Comment:
This comment isn't strictly accurate.  What the following lines do is to check for specifically the 'cinder-ceph' zaza application name.  Please consider dropping the comment.

Reply:
Modified the comment to fit what it does.